src/AnnotationThread.js

```diff
@@ -601,7 +599,7 @@ class AnnotationThread extends EventEmitter {
      * @return {void}
      */
     deleteAnnotationWithID(data) {
-        this.deleteAnnotation(data.annotationID);
+        this.deleteAnnotation(data.annotationID || data.id);
```

Comment:
This seems like something to normalize at the data layer. Where would an annotation get an `id` property?

Reply:
ah this was a mistake! clearing that up